src/main/java/ca/team2706/frc/robot/subsystems/Lift.java

```diff
@@ -306,7 +316,8 @@ public void setVelocity(int velocity) {
                 }
             }
 
-            liftMotor.configClosedLoopPeakOutput(0, 1.0); // Peak output to max (1.0).
+            liftMotor.configClosedLoopPeakOutput(FEED_FORWARD_SLOT, 1.0); // Peak output to max (1.0).
+            liftMotor.selectProfileSlot(FEED_FORWARD_SLOT, 0);
             liftMotor.set(ControlMode.Velocity, velocity);
```

Comment:
This should also use the arbitrary feed forward

Reply:
Resolved @eandr127.